fvm/errors/errors.go

```diff
@@ -114,6 +114,39 @@ func HandleRuntimeError(err error) error {
 		return NewUnknownFailure(err)
 	}
 
+	errs := []error{runErr}
+	var parentErr errors.ParentError
+	if As(err, &parentErr) {
+		errs = parentErr.ChildErrors()
+	}
+
+	codedErrs := make([]CodedError, 0)
+	for _, err := range errs {
+		coded, isUnknown := findImportantCodedError(err)
+		if isUnknown {
+			// ignore uncoded errors
+			continue
+		}
```

Comment:
This branch is incorrect. This err should either a be a `runtime.Error` a coded error  or we need to return a `NewUnknownFailure(err)`.